module/entities/helper.js

```diff
@@ -1522,14 +1523,42 @@ module.exports = class EntitiesHelper {
                   return resolve(response);
 
               } else {
-                response.result = [];
-                response["message"] = 
-                messageConstants.apiResponses.ENTITY_NOT_FOUND;
+                let groups = entitiesDetails.data[0].groups;
+
+                let targetedGroup = groups[result.entityType];
+    
+                let filterDataGroups = {
+                  "_id":targetedGroup
+                }
+    
+                entitiesDetails = await entityManagementService.entityDocuments(filterDataGroups,entityProjections);
```

Comment:
@borkarsaish65 what if an error occur? where are we checking that? can you please check this

Reply:
done